pokemonterminal/wallpaper/adapters/feh.py

```diff
@@ -1,23 +1,28 @@
 import subprocess
 import sys
-from os import system
 from pathlib import Path
 from shutil import which
 
 from . import WallpaperProvider as _WProv
 
 
 class FehProvider(_WProv):
+    compatible_wm = ['I3_PID', '_OPENBOX_PID']
+
     def change_wallpaper(path: str):
-        system(f'feh --no-fehbg --bg-fill "{path}"')
+        if (Path.home() / '.fehbg').is_file():
+            command = f'feh --bg-fill "{path}"'
+        else:
+            command = f'feh --no-fehbg --bg-fill "{path}"'
+        subprocess.run(command, check=True)
+
+    def __get_root_props() -> str:
+        return subprocess.check_output('xprop -root -notype').decode(sys.stdout.encoding)
 
     def is_compatible() -> bool:
-        if not sys.platform.startswith('linux'):
-            return False
-        root_window_prop = str(subprocess.check_output('xprop -root -notype', shell=True))
-        return which("feh") is not None and \
-            (Path.home() / '.fehbg').is_file() and \
-            any(wm_signature in root_window_prop for wm_signature in ('I3_PID', '_OPENBOX_PID'))
+        return (which("feh") is not None
```

Comment:
does `which` actually work on windows? (for when we iterate over all adapters checking `is_compatible()` for each). I thought the command was `where`.
`which` may work for you if you added the linux tools when installing git, but I'm not sure it does out of the box

Reply:
oops, read this totally wrong and didn't realize this was `shutil.which`

Reply:
Which is not a command. It's a python function and indeed works on windows.